### PR TITLE
create database SweetTweetApp and make SQL-queries

### PR DESCRIPTION
**Pull request by TsirulKate**

### CODE REVIEW

**Review by smiakchilo — COMMENTED** [review]
Hello, your PR has been noted as timely and will be assessed further on without a penalty

File: Task11/8.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+select name from USER join POST on POST.USER_ID=USER.USER_ID where datediff(current_date(),POST.CREATED_AT) = 0;
```
Comment (smiakchilo) [inline Task11/8.sql]: What about "more than 3 posts" condition?

**Review by smiakchilo — COMMENTED** [review]
The task is checked; please procceed with the next task an try to implement your project's logic in JDBC

File: Task11/8.sql (hunk 1 above)
Comment (TsirulKate) [reply Task11/8.sql]: Oach. I didn't notice. Changed it

**Review by smiakchilo — APPROVED** [review]

**Pull request merged**